.gitignore

```diff
@@ -12,3 +12,15 @@
 /captures
 .externalNativeBuild
 .cxx
+.idea/.name
```

Comment:
This can be fixed later but its annoying how some of the paths start with / and some start with no /...not your fault since it was like this before. I think it makes sense to have no / at the start of the path